unit-testing/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "unit-testing",
+  "version": "1.0.0",
+  "description": "This subfolder is the source code for the [TBD article](). The purpose is to demonstrate unit test mocks for the Azure SDK for JavaScript. These specific examples use Azure Cosmos DB.",
```

Comment:
Will this package.json be part of the sample that users see and copy or just supporting the example? If the former, I worry that things like precommit hooks etc might be confusing for users who are new

Reply:
It won't be shown in the article. I was trying to be consistent with other subfolders in this repo.